prometheus/src/main/java/io/opentelemetry/example/prometheus/PrometheusExample.java

```diff
@@ -13,45 +18,37 @@
  * The Gauge callback gets executed every collection interval.
  */
 public final class PrometheusExample {
-  private long incomingMessageCount;
-
-  public PrometheusExample(MeterProvider meterProvider) {
-    Meter meter = meterProvider.get("PrometheusExample");
-    meter
-        .gaugeBuilder("incoming.messages")
-        .setDescription("No of incoming messages awaiting processing")
-        .setUnit("message")
-        .buildWithCallback(result -> result.record(incomingMessageCount, Attributes.empty()));
-  }
-
-  void simulate() {
-    for (int i = 500; i > 0; i--) {
-      try {
-        System.out.println(
-            i + " Iterations to go, current incomingMessageCount is:  " + incomingMessageCount);
```

Comment:
I found it useful to have something printed out to the console to indicate that the thing is running.

Reply:
The new example uses the logging span exporter, so it prints out spans.  Do you think that is enough, or should I add this back in?

Reply:
Missed that! That's sufficient.